compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/generators/GeneratorExtensions.kt

```diff
@@ -48,4 +48,7 @@ open class GeneratorExtensions : StubGeneratorExtensions() {
 
     open val parametersAreAssignable: Boolean
         get() = false
+
+    open val debugInfoOnlyOnVariablesInDestructuringDeclarations: Boolean
```

Comment:
Maybe add a little explanation what this does? (E.g. to simplify further implementation in FIR)

Reply:
Done, thank you!